verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -103,7 +103,7 @@ export default class CasesController {
         try {
             const response = await axios.delete(
                 this.dataServerURL + '/api' + req.url,
-                req.body,
+                { data: req.body },
```

Comment:
why do we need the data: indirection here while all the other methods proxying just pass the req.body as is ?

Reply:
Axios quirk 🤷‍♀️   took me awhile to figure it out https://github.com/axios/axios/issues/736